python/lsst/daf/butler/registry/attributes.py

```diff
@@ -0,0 +1,114 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import annotations
+"""The default concrete implementation of the class that manages
+attributes for `Registry`.
+"""
+
+__all__ = ["DefaultButlerAttributeManager"]
+
+from typing import (
+    ClassVar,
+    Iterable,
+    Optional,
+    Tuple,
+)
+
+import sqlalchemy
+
+from ..core.ddl import TableSpec, FieldSpec
+from .interfaces import (
+    Database,
+    ButlerAttributeExistsError,
+    ButlerAttributeManager,
+    StaticTablesContext
+)
+
+
+class DefaultButlerAttributeManager(ButlerAttributeManager):
+    """An implementation of `ButlerAttributeManager` that stores attributes
+    in a database table.
+
+    Parameters
+    ----------
+    db : `Database`
+        Database engine interface for the namespace in which this table lives.
+    table : `sqlalchemy.schema.Table`
+        SQLAlchemy representation of the table that stores attributes.
+    """
+    def __init__(self, db: Database, table: sqlalchemy.schema.Table):
+        self._db = db
+        self._table = table
+
+    _META_TABLE_NAME: ClassVar[str] = "butler_attributes"
```

Comment:
In other managers I used "_META" as a prefix for things like this because they were the meta-tables that described which other tables existed.  I think I'd just use `_TABLE_NAME` and `_TABLE_SPEC` because there's only one table that's relevant for this manager.

Reply:
I missed that in renaming, will fix.